app/assets/stylesheets/modules/_family.scss

```diff
@@ -131,3 +127,66 @@
   }
 }
 
+.family-names {
+  text-align: center;
+  background-color: tint(gray, 60);
+  border-radius: $base-border-radius;
+  margin: em(10) em(80) em(30);
+  padding: em(20) 0;
+}
+
+p {
+  color: gray;
+  font-family: "Courier";
+  font-size: em(20);
+  height: em(25);
+  margin: 0 auto;
+  overflow: hidden;
+  white-space: nowrap;
+}
+
+p:nth-child(1) {
+  animation: type 9s steps(37, end);
+  animation-iteration-count: infinite;
+  -webkit-animation-delay: 2s;
+  border-right: 1px solid gray;
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.